lib/logstash/inputs/http/message_handler.rb

```diff
@@ -22,6 +22,10 @@ def validates_token(token)
       end
     end
 
+    def requires_token
```

Comment:
:open_mouth: never realized JRuby's JI also handles the reverse case (when implementing Java methods in Ruby) for method mapping fromCamel to_underscore